edweiss-app/model/school/courses.ts

```diff
@@ -30,9 +21,6 @@ export interface Course {
 	periods: CourseTimePeriod[];
 	section: Section;
 	credits: Credits;
-	newAssignments: boolean;
```

Comment:
Assignments should be handled as a sub-collection as :
1. They can grow indefinitely (the quiz structure is quite length for example)
2. They are not contingent to the Course object (I can use the object without needing the assignments)
It of course is a judgment call, but it will greatly simplify the life of anyone working with assignments.